website/docs/components/form/layout/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,13 @@
+## Anatomy
+
+![Anatomy of formLayout](/assets/components/form/layout/form-anatomy-expanded.png)
+
+| Element                           | Usage                                                                                   |
+|-----------------------------------|-----------------------------------------------------------------------------------------|
+| Title/Section title               | Optional                                                                                |
+| Description/Section description   | Optional                                                                                |
+| Section                           | May consist of multiple content types, including text, fields, and multi field groups.  |
+| Form error                        | Optional, uses the critical inline [Alert](/components/alert#status-alerts)              |
+| Field                             | Required, supports any Form component or text elements                                  |
+| Separator                         | Optional                                                                                |
+| Footer                            | Most often consists of [ButtonSet](/components/button-set) and calls to action           |
```

Comment:
[nit] We often just refer to these as actions so they're not confused with more marketing type CTAs
```suggestion 
| Footer                            | Most often consists of [ButtonSet](/components/button-set) and actions           |
```